services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchRulesSpec.groovy

```diff
@@ -173,6 +173,10 @@ class SwitchRulesSpec extends HealthCheckSpecification {
                         [
                                 installRulesAction: InstallRulesAction.INSTALL_ROUND_TRIP_LATENCY,
                                 cookie            : Cookie.ROUND_TRIP_LATENCY_RULE_COOKIE
+                        ],
+                        [
+                                installRulesAction: InstallRulesAction.INSTALL_UNICAST_VXLAN,
+                                cookie            : Cookie.VERIFICATION_UNICAST_VXLAN_RULE_COOKIE
```

Comment:
Again, this is not expected to work on _all_ switches.

Reply:
thx, fixed
needed `assumeFalse` was added